### ISSUE TITLE
Authorize access to select repositories only?

### ISSUE THREAD

**Issue by dpo** [google/github-issue-mover#116]
Is it possible to only authorize access to select repositories? As it stands, `github-issue-mover` requests access to absolutely everything. Thanks.

**Comment by tjomamokrenko** [comment]
:+1:

**Comment by mlissner** [comment]
This is particularly bad when you belong to several organizations and it requests access to all of them. It also requests access to settings, deploy keys and webhooks, which seem unnecessary and possibly intrusive. Alas, I can't use this as it stands.

**Comment by jafflerbach** [comment]
Agreed - I belong to 4 organizations, I want to use this application on just one of those. I can't justify giving access to this application for all 4 organizations I am a part of. Especially with these permissions:

![image](https://cloud.githubusercontent.com/assets/6842510/24768563/e260346e-1ab7-11e7-85bc-0d8ee5ac9326.png)

**Comment by mlissner** [comment]
Deploy keys!
